library/src/hcc_detail/hipblas.cpp

```diff
@@ -16832,6 +16841,480 @@ catch(...)
     return exception_to_hipblas_status();
 }
 
+// gelsBatched
+hipblasStatus_t hipblasSgelsBatched(hipblasHandle_t    handle,
+                                    hipblasOperation_t trans,
+                                    const int          m,
+                                    const int          n,
+                                    const int          nrhs,
+                                    float* const       A[],
+                                    const int          lda,
+                                    float* const       B[],
+                                    const int          ldb,
+                                    int*               info,
+                                    int*               deviceInfo,
+                                    const int          batchCount)
+try
+{
+    if(info == NULL)
+        return HIPBLAS_STATUS_INVALID_VALUE;
+    else if(m < 0)
+        *info = -2;
+    else if(n < 0)
+        *info = -3;
+    else if(nrhs < 0)
+        *info = -4;
+    else if(A == NULL)
+        *info = -5;
+    else if(lda < m)
+        *info = -6;
+    else if(B == NULL)
+        *info = -7;
+    else if(ldb < m || ldb < n)
+        *info = -8;
+    else if(info == NULL)
+        *info = -9;
```

Comment:
I suppose these checks on `info` can be removed, since they should be unreachable (and would segfault).

Reply:
lol yeah, a pretty bad two lines of code just looking at it now.